src/components/AuthStoreSetter.js

```diff
@@ -0,0 +1,28 @@
+import _ from 'lodash/fp'
+import { useEffect } from 'react'
+import { useAuth } from 'react-oidc-context'
+import { processUser, reloadAuthToken } from 'src/libs/auth'
+import { useOnMount } from 'src/libs/react-utils'
+import { authStore } from 'src/libs/state'
+
+
+const AuthStoreSetter = () => {
+  const auth = useAuth()
+  useOnMount(() => authStore.update(_.set(['authContext'], auth)))
+
+  useEffect(() => {
```

Comment:
I did it this way because each of the callback registrations returns a cleanup function which I wanted to hook into `useEffect` cleanup. There may be a way to do it in 1 useEffect block though. (And need to investigate the console errors you noted above too.)

Reply:
Ah, I see.

Reply:
The recommendation on this is to combine into a single `useEffect`. Along these lines, but this hasn't been tested yet.

````
  useEffect(() => {
    const cleanupFns = [
      auth.events.addAccessTokenExpiring(reloadAuthToken),
      auth.events.addUserLoaded(processUser),
      auth.events.addUserUnloaded(processUser)
    ]
    return () => _.forEach(fn => fn(), cleanupFns)
  }, [auth])